src/Entity/Committee.php

```diff
@@ -205,6 +205,13 @@ public function getDescription(): string
         return $this->description;
     }
 
+    public function setDescription(string $description): self
```

Comment:
Why is this setter needed?

Reply:
Because if we don t have a setter we can't modify the description with an admin form

Reply:
No, take a look at https://github.com/EnMarche/en-marche.fr/blob/develop/src/Entity/EntityCrudTrait.php.

Reply:
😱